src/Illuminate/Encryption/Encrypter.php

```diff
@@ -184,12 +194,11 @@ public function decryptString($payload)
      *
      * @param  string  $iv
      * @param  mixed  $value
-     * @param  string  $tag
      * @return string
      */
-    protected function hash($iv, $value, $tag = '')
+    protected function hash($iv, $value)
     {
-        return hash_hmac('sha256', $tag.$iv.$value, $this->key);
+        return hash_hmac('sha256', $iv.$value, $this->key);
```

Comment:
Note that the only change here and in `validMac` is reverting the unnecessary additions of https://github.com/laravel/framework/pull/38190/files